core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionPolicyManager.java

```diff
@@ -162,10 +162,10 @@ public TransmissionPolicyStateFetcher getTransmissionPolicyState() {
 
     private synchronized void doSuspend(TransmissionPolicy policy, long suspendInSeconds) {
         try {
-            if (policy == TransmissionPolicy.UNBLOCKED ) {
+            if (policy == TransmissionPolicy.UNBLOCKED) {
                 return;
             }
-                       
+
             Date date = Calendar.getInstance().getTime();
             date.setTime(date.getTime() + 1000 * suspendInSeconds);
```

Comment:
Can we add explicit braces here.  date.setTime(date.getTime() + (1000 * suspendInSeconds))
I know that multiplication takes precedence over addition but I would still prefer to be explicit to avoid any malfunctions in situations.